fairlearn/reductions/_grid_search/grid_search.py

```diff
@@ -79,7 +79,7 @@ def accumulate_integer_grid(self, index, max_val):
                 self.accumulate_integer_grid(index + 1, max_val - abs(current_value))   # noqa: E501
 
 
-class GridSearch(Reduction):
+class GridSearch(BaseEstimator, ClassifierMixin, RegressorMixin):
     """Estimator to perform a grid search given a blackbox estimator algorithm.
```

Comment:
This should be `BaseEstimator` and `MetaEstimatorMixin`.

As with `ExponentiatedGradient`, it would be to set `_estimator_type` and `score` functionality according to our base estimator... but happy to leave that for a separate PR.